packages/contracts/contracts/PCV.sol

```diff
@@ -16,7 +16,7 @@ contract PCV is IPCV, Ownable, CheckContract, SendCollateral {
     // --- Data ---
     string constant public NAME = "PCV";
 
-    uint256 constant public BOOTSTRAP_LOAN = 10 * 10**18; 
+    uint256 constant public BOOTSTRAP_LOAN = 10**26; // 100M thUSD
```

Comment:
This went from 10M to 100M. is that intended?

Reply:
I think it went from 100 to 100M

Reply:
Sorry yes, 10 -> 100M